go/host/enclave/state.go

```diff
@@ -157,6 +157,12 @@ func (s *StateTracker) OnPromoted() {
 	s.enclaveIsSequencer = true
 }
 
+func (s *StateTracker) OnDemoted() {
```

Comment:
I'd rename this to "SetActiveSequencer(bool)" - for clarity.

"OnDemoted" - sounds like it's doing a bunch of stuff, and there is some dynamic logic